core/src/main/java/bisq/core/offer/OpenOfferManager.java

```diff
@@ -227,7 +227,10 @@ public void onUpdatedDataReceived() {
     }
 
     private void cleanUpAddressEntries() {
-        Set<String> openOffersIdSet = openOffers.getList().stream().map(OpenOffer::getId).collect(Collectors.toSet());
+        Set<String> openOffersIdSet = openOffers.getList().stream()
+                .filter(openOffer -> !hasOfferSharedMakerFee(openOffer))
```

Comment:
> .filter(openOffer -> !hasOfferSharedMakerFee(openOffer))

BUG: This change causes the reserved amount for cloned offers to be flipped to AVAILABLE whenever Bisq is starte up.  Then it can be spent and the offers have an invalid maker fee tx.  Reverting that line makes the bug go away.

Reply:
BUG: Create an offer. Clone it.  Remove the 1st offer and the clone.  Go to funds Send funds to send your security deposit.  It does not appear in available funds.  Check reserved funds, not there either.  The security deposit is not accessible any more.  It only becomes available after shutting down and restarting Bisq, when cleanUpAddressEntries gets triggered again.

Reply:
I don't understand why that should cause an issue. The `.filter(openOffer -> !hasOfferSharedMakerFee(openOffer))` filters out any offers which have shared maker fee.

Reply:
Fixed with https://github.com/bisq-network/bisq/pull/6675/commits/60fad3ab6cccd27f5d881848386f6b52e98fc31d